### DISCUSSION TITLE
CSS Update in Permissions

### DISCUSSION THREAD

**Discussion by erondpowell** [directus/directus#12944] [Archive]
The checks are still green...
Also, I was working with Ben and saw that his instance had a purple check with a subtle green background.

<img width="1728" alt="Screen Shot 2022-04-21 at 6 47 17 PM" src="https://user-images.githubusercontent.com/62306630/164568802-7d90ff3b-c512-4d4c-aea9-634a2f937c2d.png">

**Comment by azrikahar** [comment]
It should still be green if I'm not mistaken, at least it is from my end as well. Perhaps @benhaynes has a custom CSS that changes `--v-icon-color` or `--success` variable? (and maybe since the subtle green uses `--v-icon-color-hover`, my initial guess would be it's just that it wasn't overridden via custom CSS, thus causing this confusion)

On that note, it's an interesting question whether this green check mark should be tied to brand color! On one hand, think green could still be fitting as that's a generally known "success" or "turned on" color, but on the other hand since we're already using icons to distinguish them, I reckon it wouldn't be confusing if it's not green.

**Comment by rijkvanzanten** [comment]
IMO the checks indicate a "success" state, and therefore should use `--success` rather than `--primary` 👍🏻

**Reply by benhaynes** [reply]
Yes, in my demo it was half green and half purple... probably my CSS overrides (🤦 ). These should use green/success.
